src/views/RecIM/views/Team/index.js

```diff
@@ -1,95 +1,104 @@
-        <MatchList matches={[{ activityID: '1', ID: '789' }]} />
-        {/* else "no schedule yet set" */}
-        <Typography variant="body1" paragraph>
-          Games have not yet been scheduled.
-        </Typography>
-      </CardContent>
-    </Card>
-  );
+  useEffect(() => {
+    const loadTeamData = async () => {
+      setLoading(true);
+      setTeam(await getTeamByID(teamID));
+      setLoading(false);
+    };
+    loadTeamData();
+  }, [teamID]);
 
   if (loading) {
     return <GordonLoader />;
   } else if (!profile) {
     // The user is not logged in
     return <GordonUnauthorized feature={'the Rec-IM page'} />;
   } else {
+    let teamHeader = (
+      <Card>
+        <CardContent>
+          <Grid container direction="column">
+            <Grid item container direction="column" alignItems="center">
+              <Grid item>
+                <Breadcrumbs aria-label="breadcrumb">
+                  <LinkRouter
+                    className="gc360_text_link"
+                    underline="hover"
+                    color="inherit"
+                    to={'/recim'}
+                  >
+                    <HomeIcon sx={{ mr: 0.5 }} fontSize="inherit" />
+                    Rec-IM Home
+                  </LinkRouter>
+                  <LinkRouter
+                    className="gc360_text_link"
+                    underline="hover"
+                    color="inherit"
+                    to={`/recim/activity/${activityID}`}
+                  >
+                    Activity Name
+                  </LinkRouter>
+                  <Typography color="text.primary">Team Name</Typography>
+                </Breadcrumbs>
+              </Grid>
+              <hr className={styles.recimNavHeaderLine} />
+            </Grid>
+            <Grid item container direction="row" alignItems="center" columnSpacing={4}>
+              <Grid item>
+                <img src={''} alt="Team Icon" width="85em"></img>
+              </Grid>
+              <Grid item xs={8} md={5}>
+                <Typography variant="h5" className={styles.teamTitle}>
+                  Team Name
+                </Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+        </CardContent>
+      </Card>
+    );
+
+    let rosterCard = (
+      <Card>
+        <CardHeader title="Roster" className={styles.cardHeader} />
+        <CardContent>
+          <ParticipantList participants={team.Participant} />
+        </CardContent>
+      </Card>
+    );
+
+    let scheduleCard = (
+      <Card>
+        <CardHeader title="Schedule" className={styles.cardHeader} />
+        <CardContent>
+          {team.Match?.length ? (
+            <MatchList matches={team.Match} activityID={team.ActivityID} />
+          ) : (
+            <Typography variant="body1" paragraph>
+              No matches scheduled at this time!
+            </Typography>
+          )}
+        </CardContent>
+      </Card>
+    );
+
```

Comment:
Is there any added benefit of making these variables if they're only ever used once?